awx/ui/client/src/templates/prompt/prompt.controller.js

```diff
@@ -181,7 +181,8 @@ export default [ 'ProcessErrors', 'CredentialTypeModel', 'TemplatesStrings',
                         }
                         vm.steps.preview.tab.order = order;
                         vm.steps.preview.tab._disabled = vm.readOnlyPrompts ? false : true;
-                        modal.show(strings.get('prompt.PROMPT'));
+                        let templateName = vm.promptDataClone.templateName;
+                        modal.show(strings.get('prompt.PROMPT'), null, templateName);
```

Comment:
Job template names reflect input from other users, so let's apply a `$filter('sanitize')` to `templateName` here.